store/tx_manager.go

```diff
@@ -471,32 +471,31 @@ func (txm *EthTxManager) handleConfirmed(
 }
 
 func (txm *EthTxManager) handleUnconfirmed(
-	tx *models.Tx,
-	txat *models.TxAttempt,
+	latestTxAttempt *models.Tx,
+	txAttempt *models.TxAttempt,
 	blkNum uint64,
 ) (*TxReceipt, error) {
-	// Only handle the latest transaction attempt, not the remainder
-	if tx.Hash != txat.Hash {
+	if txAttempt.Hash != latestTxAttempt.Hash {
```

Comment:
I think this might all be more clear if this line were a method like `tx.IsLatestAttempt(txAttempt)`. Thoughts?